src/app/api/api_v1/endpoints/organizations.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2024, Pyronear.
+
+# This program is licensed under the Apache License 2.0.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0> for full license details.
+
+
+from typing import List, cast
+
+from fastapi import APIRouter, Depends, HTTPException, Path, Security, status
+
+from app.api.dependencies import get_jwt, get_organization_crud
+from app.crud import OrganizationCRUD
+from app.models import Organization, UserRole
+from app.schemas.login import TokenPayload
+from app.schemas.organizations import OrganizationCreate
+from app.services.telemetry import telemetry_client
+
+router = APIRouter()
+
+
+@router.post("/", status_code=status.HTTP_201_CREATED, summary="Register a new organization")
+async def register_organization(
+    payload: OrganizationCreate,
+    organizations: OrganizationCRUD = Depends(get_organization_crud),
+    token_payload: TokenPayload = Security(get_jwt, scopes=[UserRole.ADMIN]),
+) -> Organization:
+    telemetry_client.capture(token_payload.sub, event="organization-create", properties={"device_login": payload.name})
```

Comment:
What I usually do is that I add as properties the target when it's a GET/UPDATE/DELETE. I'd suggest removing this property, especially because this isn't a device_login